Examples/laser_acceleration/laser_acceleration_PICMI.py

```diff
@@ -103,9 +103,10 @@
 # Set the ionization for the species number 1 (Argon)
 # and place the created electrons into the species number 2 (electron)
 if picmi.codename != 'warpx':
```

Comment:
Since you will be adding this capability to WarpX, this `if` check will presumably be removed (either in this PR or a subsequent one). And the same below.